chapter2/tests/t4/chapter2-l2-t4-files-no-extensions.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# The scripts returns list of files without extension.
+set -eou pipefail
+
+IFS=$'\n'
+
+dir="${1}"
+
+if [[ ! -e "${dir}" || ! -d "${dir}" ]]; then
+  echo "The '${dir}' does not exist or it is not a directory." >&2
+  exit 1
+fi
+
+for file in $(find "${dir}" -type f 2> /dev/null); do
```

Comment:
Do we really need to send stderr output to `/dev/null` here? I don't think that it is good to hide errors

Reply:
Not sure, but removed that.